andela/boondocks-bn-backend
# #169257384 Setup Sequelize ORM

#### What does this PR do?
Sets up sequalize ORM, the migrations and add all the configurations for sequalize. Creates a dummy user model to test the configurations.
#### Description of Task to be completed?
- [x] set up Postgres database
- [x]  initialize sequalize
-  [x] add babel to transpile sequalize files during migrations
- [x]  Describe how to set up the database on README.md file
- [x]  create a dummy user model to test the configurations
#### How should this be manually tested?
- clone the repository
- run `npm install` to install dependencies
- check `README.md` on how to set up the database
- then run `npm run migrate` to check if the migration works
- go to the database and check if the table user was created

#### Any background context you want to provide?
- This PR sets up the sequalize which will be used for the main project, no features implemented yet
#### What are the relevant pivotal tracker stories?
- [#169257384](https://www.pivotaltracker.com/n/projects/2407417/stories/169257384)
#### Screenshots 
- N/A
